metashare/api/jobs.py

```diff
@@ -477,46 +478,48 @@ def populate_github_users(repository):
 def submit_review(*, user, task, data):
     try:
         review_sha = None
+        org = data["org"]
```

Comment:
I'll pull `data["notes"]` out similarly; it'll clean up legibility to not be doing dict lookup constantly.